comm_sbs_bqctrl.py

```diff
@@ -3728,7 +3744,7 @@ def main():
                 po.i32key = 0x36720414
         smart_battery_system_sealing(po.sealstate, vals, po)
     else:
-        raise NotImplementedError("Unsupported command.")
+        raise NotImplementedError("Unsupported or missing command\n  run\ncomm_sbs_bqctrl help")
```

Comment:
The argument would have to be `--help`. Also, is this code even reachable? I thought of it more like an assert.
Please remove this change.

Reply:
I see. Then the parser should be set to require the action argument.
It will make this script require python 3.7, but we're only supporting 3.8+ anyway.

Reply:
I've committed changes